test/mocha/core/utils.js

```diff
@@ -1402,4 +1404,22 @@ describe("utils", function() {
       expect(res).toEqual("asdf")
     })
   })
+
+  describe("generateCodeVerifier", function() {
+    it("should generate a value of at least 43 characters", () => {
+      const codeVerifier = generateCodeVerifier()
+
+      // Source: https://tools.ietf.org/html/rfc7636#section-4.1
+      expect(codeVerifier.length).toBeGreaterThanOrEqualTo(43)
+    })
+  })
+
+  describe("createCodeChallenge", function() {
+    it("should hash the input using SHA256 and output the base64 url encoded value", () => {
```

Comment:
I know hard-coding the 2 values is not the best, but I wanted to make sure the `code_challenge` was indeed the SHA256 of the `code_verifier` in base64 url format.

Reply:
No problem, but let's add a comment next to the values so it's clear what they are?